types/chai-datetime/chai-datetime-tests.ts

```diff
@@ -1,9 +1,9 @@
-import chai = require("chai");
 import chaiDateTime = require("chai-datetime");
 
-chai.use(chaiDateTime);
-const expect = chai.expect;
-const assert = chai.assert;
+import("chai").then(chai => chai.use(chaiDateTime));
+
+const expect: Chai.ExpectStatic = {} as any;
+const assert: Chai.AssertStatic = {} as any;
```

Comment:
For all of these `const expect: ...`, I assume these are here because they are in CJS packages which cannot import chai anymore?

I would actually suggest writing these like:

```ts
declare const expect: Chai.ExpectStatic;
declare const assert: Chai.AssertStatic;
```

Rather than using the cast. I personally find the intent clearer for DT tests where declaring things sort of means "I don't care I'm just declaring this to write the tests".